internal/websocket/options.go

```diff
@@ -273,45 +273,45 @@ func MaxMessageBytes(bytes int64) Option {
 
 // AddMessageListener adds a message listener to the WS connection.
 // The listener will be called for every message received from the WS.
-func AddMessageListener(listener event.MsgListener, cancel ...*event.CancelFunc) Option {
+func AddMessageListener(listener event.MsgListener, cancel ...*func()) Option {
```

Comment:
I don't think this is the right way to achieve this.  Can we not explicitly cast the result to a `func()` afterwards?

Reply:
for sure! actually we don't need too since it'll be implicitly casted.

I just figured we wanted to follow what you did for the eventor https://github.com/xmidt-org/eventor/pull/7/files (replacing its `CancelFunc` with `func()`)

Reply:
Ok, I see the trend is to be more consistent.  Great!